Libs/vtkSegmentationCore/vtkSegmentation.h

```diff
@@ -304,7 +308,63 @@ class vtkSegmentationCore_EXPORT vtkSegmentation : public vtkObject
   /// Invalidate (remove) non-master representations in all the segments if this segmentation node
   void InvalidateNonMasterRepresentations();
 
-// Conversion related methods
+// Merged labelmap utility functions
+
+  /// Returns true if the binary labelmap representation is merged
+  bool IsBinaryLabelmapMerged(std::string segmentID);
+
+  void GetMergedSegmentIDsForRepresentation(vtkSegment* segment, std::string representationName,
```

Comment:
Please add documentation. It's far from trivial what is a merged segment ID list, what are shared segments, and what is a main segment. Thanks